### ISSUE TITLE
install.sh: Create an env var that lets the user specify a hostname, and an env var to auto-recover a Sandcats domain

### ISSUE THREAD

**Issue by paulproteus** [sandstorm-io/sandstorm#1029]
This paves the way for fully automated installs.

Example invocation:

```
sudo SANDCATS_HOSTNAME=example SANDCATS_RECOVERY_TOKEN=80f5b94de13e6fcca37eacee2179c3695a4fea94 bash install.sh
```

**Comment by paulproteus** [comment]
Self-assigning to minimize the chance I forget about it. If you need to take the ticket from me, then just leave a comment and I can de-assign.

**Comment by paulproteus** [comment]
Fixed in 6b5351c4a58c29e9bf3ad6d16bae27895aeaad38 so closing now.
